rt-shell/objects/obj_shell/Draw_64.gml

```diff
@@ -2,6 +2,7 @@ if (isOpen) {
 	draw_set_font(consoleFont);
 	// pre-calculate one "em" of height
 	var lineHeight = string_height("M");
+	var lineWidth = string_width("M");
```

Comment:
this is not actually an accurate variable name, `characterWidth` might be better? Or maybe we should change both of these variables to `emHeight` and `emWidth` to be more uniform. Also update the comment above saying we're just calculating height